libs/network/include/network/muddle/rpc/server.hpp

```diff
@@ -1,154 +0,0 @@
-#pragma once
```

Comment:
When did these get reintroduced have we had a merge blunder somewhere?

Reply:
You may have introduced it in commit 1bc800ac

Reply:
Oh the nostalgia - that was when it originally came into `master` I was pretty sure that I binned / moved these files around with Muddle V2. Oh well you got the stragglers! 😁